plugins/source/aws/policies/README.md

```diff
@@ -1,78 +1,9 @@
 # CloudQuery Policies
 CloudQuery SQL Policies for AWS
 
-## Policies and Compliance Frameworks Available
+## Deprecated:
 
-- [AWS CIS V1.2.0](./cis_v1.2.0/policy.sql)
-- [AWS PCI DSS v3.2.1](./pci_dss_v3.2.1/policy.sql)
-- [AWS Foundational Security Best Practices](./foundational_security/policy.sql)
-- [AWS Public Egress](./public_egress/policy.sql)
-- [AWS Publicly Available](./publicly_available/policy.sql)
-- [AWS Unused Resources](./unused_resources/policy.sql)
+CloudQuery SQL Policies for AWS has been deprecated all queries have been moved to the `/policies_v1` directory. Going forward all CloudQuery Policies will continue to be available in the [CloudQuery Hub](https://hub.cloudquery.io/addons/transformation). The Policies listed via the Hub have many benefits including:
+- Built on top of DBT for ease of extensibility and integration with other tools
+- Multi database support (PostgreSQL, BigQuery, Snowflake)
```

Comment:
```suggestion
- Multi-database support (PostgreSQL, BigQuery, Snowflake)
```

Reply:
completed in https://github.com/cloudquery/cloudquery/pull/15487/commits/26eb52212f571662ec1fe7c352a7fc16448fdb23
and 
https://github.com/cloudquery/cloudquery/pull/15487/commits/b3a2a6e137cb470e07c2648c060bdd8dbbfb4d1f